.test-infra/jenkins/job_PostCommit_Java.groovy

```diff
@@ -40,7 +40,7 @@ PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java', 'Run Java PostCommit'
       steps {
         gradle {
           rootBuildScriptDir(commonJobProperties.checkoutDir)
-          tasks(':javaPostCommit')
+          tasks(':installVendoredGrpc', ':javaPostCommit')
```

Comment:
Jenkins does not respect this until we tell Jenkins to do so. (That might affect other PRs?)